### ISSUE TITLE
Implement Cascading Menu in "Open in Terminal (Preview)" Option inside Context Menu

### ISSUE THREAD

**Issue by roywillynson** [microsoft/terminal#15125]
# Description of the new feature/enhancement

**Problem:** I only can open directories in Windows Terminal (Preview) through my default shell (Ubuntu, through WSL). I also should like to open it from another shell like PowerShell, Git Bash and so on. I think it would be good to add a sub/cascading menu to "Open in Terminal (Preview)" Option inside Context Menu.

**Suggestion:** The "Open in Terminal (Preview)" Option inside Context Menu could display a cascading menu inside with all shells by default.

# Proposed technical implementation details (optional)

**Example in Windows 11:**

**Before:**Current Context Menu without Cascading Menu when right-clicking a directory.
![Context Menu without Cascading Menu](https://user-images.githubusercontent.com/65576262/230281109-e4dafbae-02a1-4e97-8211-3e8918573330.png)

**After:** Context Menu with Cascading Menu when right-clicking a directory.

![Context Menu with Cascading Menu](https://user-images.githubusercontent.com/65576262/230284021-b15dfb50-4fcf-4757-9a5a-cd1c40cc8c71.png)

*This is a suggestion.*

**Comment by zadjii-msft** [comment]
Thanks for the suggestion! This is actually already being tracked by another issue on our repo - please refer to #6111 for more discussion.

/dup #6111

**Comment by roywillynson** [comment]
Ok, I stay tuned to changes and I hope you can implement it soon😅.
